editor/store/effects.js

```diff
@@ -471,6 +471,9 @@ export default {
 		speak( message, 'assertive' );
 	},
 	INITIALIZE_META_BOX_STATE( action, store ) {
+        if ( ! has( window, 'postboxes.add_postbox_toggles' ) ) {
+            return;
+        }
 		// Allow toggling metaboxes panels
 		if ( some( action.metaBoxes ) ) {
```

Comment:
Should we combine the check with this line instead, we still need what's next.

Also, in which context, it's not available? Asking because the fix could be different, maybe the meta boxes as a whole are not valid in some contexts and we should find a way to extract those to `edit-post` instead.

Reply:
Meta boxes are not available on WPCOM, so as whole the action is not needed. I tried disabling at the sidebar, but even when I didn't include the `<Metabox>` component this error still occurred. 

As far as combining, it seems like we could include another exit `if ( ! some( action.metaBoxes ) )` just return and that would cover the additional parts.